sys/linux/auto.txt

```diff
@@ -5824,6 +7056,97 @@ peer_policy$auto_netlink [
 	WGPEER_A_PROTOCOL_VERSION		nlattr[WGPEER_A_PROTOCOL_VERSION, int32]
 ] [varlen]
 
+perf_event$auto_record {
+	bpf_cookie	intptr
+}
+
+perf_event_attr$auto_record {
+	type				int32
+	size				int32
+	config				intptr
+	perf_event_attr_3		perf_event_attr_3$auto_record
+	sample_type			intptr
+	read_format			intptr
+	disabled			intptr
```

Comment:
https://elixir.bootlin.com/linux/v6.11/source/include/uapi/linux/perf_event.h#L414

Note that these are bitfields.

See https://github.com/google/syzkaller/blob/master/docs/syscall_descriptions_syntax.md#ints